ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vxlan-interface.j2

```diff
@@ -4,30 +4,54 @@
 
 ### VXLAN Interface Summary
 
-#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.source_interface }}
+#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.vxlan.source_interface }}
 
-#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan_udp_port }}
+#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan.udp_port }}
 
-#### VLAN to VNI Mappings
+{% if vxlan_tunnel_interface.Vxlan1.vxlan.virtual_router_encapsulation_mac_address is defined %}
```

Comment:
Please use `arista.avd.defined` filter as it run deep variable validation

Reply:
Updated
```jinja
{% if vxlan_tunnel_interface.Vxlan1.vxlan.virtual_router_encapsulation_mac_address is arista.avd.defined %}
```